src/exml_query.erl

```diff
@@ -44,7 +52,8 @@ path(Element, Path) ->
 path(#xmlel{} = Element, [], _) ->
     Element;
 path(#xmlel{} = Element, [{element, Name} | Rest], Default) ->
-    Child = subelement(Element, Name), % may return undefined
+    Child = subelement(Element,
```

Comment:
Why exactly this was split into two lines? I think the old line length is less that 100 chars.

Reply:
This one is very strange. I checked out `master` branch of this repo on my computer, modified the `rebar.config` just like you did and run `rebar3 format` and it doesn't change the line.

Reply:
I did the same @elbrujohalcon end the result is the same as before for me. Do you think it makes a difference what OS or Erlang version is used when running the formatter? I'm on newest Mac OS X and Erlang OTP 22.2

Reply:
🤔 
My Erlang is `Erlang/OTP 21 [erts-10.3.5.3] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]`.
I can try with 22.2 later.